doc/manual/install-judgehost.rst

```diff
@@ -19,6 +19,10 @@ System requirements
 * A TCP/IP network which connects the DOMserver and the judgehosts.
   The machines only need HTTP(S) access to the DOMserver.
 
+For consistent timing, we strongly recommend to only run a single
```

Comment:
We have the section about running multiple judgehosts, I think we should also link there or better have a short mention here and link to that section and discuss there the different tradeoffs?

Reply:
Updated.